libs/auth/src/java_card.ts

```diff
@@ -314,13 +313,7 @@ export class JavaCard implements Card {
         // Okay if this is larger than data.length as .subarray() automatically caps
         MAX_DATA_OBJECT_SIZE_BYTES * i + MAX_DATA_OBJECT_SIZE_BYTES
       );
-      await this.putData(
-        objectId,
-        Buffer.concat([
-          constructTlv(PUT_DATA.TAG_LIST_TAG, objectId),
-          constructTlv(PUT_DATA.DATA_TAG, chunk),
-        ])
-      );
+      await this.putData(objectId, chunk);
```

Comment:
I'd forgotten that `putData` internally handles wrapping the data

Reply:
Do we need a regression test for this bug?

Reply:
Yes we do! I'm incorporating this into the tests I'm still working on for the JavaCard class